src/main/markdown/doc/latest/RefCommandLineTools.md

```diff
@@ -47,7 +47,7 @@ Use the files generated by these scripts as a starting point for building your o
 | ------------------------ | ---------- |
 | -[no]overwriteFiles      | Overwrite any existing files. (defaults to OFF)
 | -[no]ignoreExistingFiles | Ignore any existing files; do not overwrite. (defaults to OFF)
-| -templates               | Specifies the template(s) to use (comma separeted). Defaults to 'sample,ant,eclipse,readme'
+| -templates               | Specifies the template(s) to use (comma separated). Defaults to 'sample,ant,eclipse,readme'
```

Comment:
Comes from https://github.com/gwtproject/gwt/blob/78215ac9bd69a725375e5b3a36b83ed8f58ee5ad/user/src/com/google/gwt/user/tools/WebAppCreator.java#L400 , but maybe OK to fix here and not in the deprecated Java class?

Reply:
We don't generate the docs at this time, so changing it here first seems fine to me.